webapp/components/ContentMenu/ContentMenu.vue

```diff
@@ -191,6 +191,15 @@ export default {
               icon: 'user-times',
             })
           }
+          if (this.isAdmin === true) {
+            routes.push({
+              label: this.$t(`settings.deleteUserAccount.name`),
+              callback: () => {
+                this.$emit('delete', this.resource)
+              },
+              icon: 'trash',
+            })
+          }
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-05-28T10:31:50Z" title="Thursday, May 28th 2020, 12:31:50 pm +02:00">May 28, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -156,6 +156,15 @@ export default {
             path: '/settings',
             icon: 'edit',
           })
+          if (this.isAdmin === true) {
+            routes.push({
+              label: this.$t(`settings.deleteUserAccount.name`),
+              callback: () => {
+                this.$emit('delete', this.resource)
+              },
+              icon: 'trash',
+            })
+          }
```


Admins shouldn’t be able to delete their own profile on here. Instead they should be forced to go to their settings.
Here they even see or have no options to let their contributions and comments undeleted.
That’s my opinion so far …

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-08-14T15:50:40Z" title="Friday, August 14th 2020, 5:50:40 pm +02:00">Aug 14, 2020</time>_

---


fixed